nucypher/blockchain/eth/clients.py

```diff
@@ -250,6 +250,21 @@ def sign_message(self, account: str, message: bytes) -> str:
         """
         return self.w3.eth.sign(account, data=message)
 
+    def get_signing_key(self, account):
```

Comment:
I do not think we want to expose local key signing here, since it's handled by the client's keyring.  We apply this logic below to `EthereumTester` because it does not expose the `signTransaction` RPC endpoints, but wed like to test our code using those calls.